encoder/vaapiencoder_vp8.cpp

```diff
@@ -75,6 +75,10 @@ void VaapiEncoderVP8::resetParams()
 {
     //5 times compress ratio
     m_maxCodedbufSize = width() * height() * 3 / 2;
+    //when the resolution of video is very small such as 96x96 and buffer plus the header of vp8,
+    //the size of actual output buffer maybe be bigger the m_maxCodedbufSize.
+    //So plusing 4096 avoids the situation.
+    m_maxCodedbufSize += 0x1000;
```

Comment:
Hi Daniel:
     Yeah, if the context is very complex and the quantization value (q_index for vp8 codec) is very small, the coded slice data will be very close to the limitation value width() * height() * 3 / 2. And the coded bitstream (slice_data + frame headers) will more than width() * height() * 3 / 2. 
  Any resolution may encounter this issue. But it more likely happens on small resolution. Because frame header with fixed size independent on resolution, but smaller resolution with smaller slice_data. As Guangxin mentioned, here 0x1000 is presented frame header size, it is large enough.